client/my-sites/post-relative-time-status/index.jsx

```diff
@@ -96,10 +97,22 @@ class PostRelativeTime extends React.PureComponent {
 			statusText = this.props.translate( 'pending review' );
 			statusClassName += ' is-pending';
 		} else if ( status === 'future' ) {
-			const scheduledTime = this.props.moment( this.props.post.date ).fromNow();
-
-			statusText = this.props.translate( 'scheduled %(scheduledTime)s', {
-				comment: '%(scheduledTime)s is a future human time, for example "in 3 days"',
+			const moment = this.props.moment;
+			const scheduledDate = moment( this.props.post.date );
+			const now = moment();
+			const scheduledTime =
+				scheduledDate.diff( now, 'days' ) <= 7
+					? scheduledDate.calendar()
+					: scheduledDate.format( 'LL' ) +
+					  ' ' +
+					  this.props.translate( 'at', {
+							context: 'scheduled for 21 April at 02:00',
+					  } ) +
+					  ' ' +
+					  scheduledDate.format( 'LT' );
+
+			statusText = this.props.translate( 'scheduled for %(scheduledTime)s', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **1** times:
`translate( 'scheduled %(scheduledTime)s' )` *ES Score: 7*